roles/tomcat/tasks/main.yml

```diff
@@ -19,6 +19,11 @@
 - name: Add tomcat bin to path environment
   lineinfile: dest={{ env_path }} state=present backrefs=yes regexp='PATH=(["]*)((?!.*?{{ tomcat_bin }}).*?)(["]*)$' line="PATH=\1\2:{{ tomcat_bin }}\3"
```

Comment:
Можна просто додати в той файл
PATH+="path/to/tomcat"


In roles/tomcat/tasks/main.yml<https://github.com/IF-061-DevOps/OMS-ansible/pull/88#discussion_r59972113>:

> @@ -19,6 +19,11 @@
>  - name: Add tomcat bin to path environment
>    lineinfile: dest={{ env_path }} state=present backrefs=yes regexp='PATH=(["]*)((?!.*?{{ tomcat_bin }}).*?)(["]*)$' line="PATH=\1\2:{{ tomcat_bin }}\3"


Але працю?...

-
You are receiving this because you commented.
Reply to this email directly or view it on GitHub<https://github.com/IF-061-DevOps/OMS-ansible/pull/88/files/e585f912960ab449c5eaa36fe00074c26139f1ff#r59972113>